examples/backups/vmi/06-restore-in-place.sh

```
@@ -0,0 +1,49 @@
+  name: restore-in-place-test
+  namespace: ${NAMESPACE}
+spec:
+  backupRef:
+    name: test-backup
+  targetApplicationRef:
+    apiGroup: apps.cozystack.io
+    kind: VMInstance
+    name: test
+  options:
+    failIfTargetExists: true # if true, restore will fail when the target resource already exists
+    keepOriginalPVC: true # renames original VMI PVC before restore to `<name>-orig-<hash>`, only for in-place restore
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)